### PR TITLE
fix: rename internal types

### PR DESCRIPTION
**Pull request by Fabio1988**
Try to cleanup and follow naming conventions

### CODE REVIEW

File: packages/types/src/status.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+import type { ControllerConnectionDTO, DeviceControlDTO, DeviceWorkerDTO } from '@rotom/connections';
 
-export { ScannerConnectionDTO, MitmControlDTO, MitmWorkerDTO };
+export { ControllerConnectionDTO, DeviceControlDTO, DeviceWorkerDTO };
 
 export type WorkerDTO = {
+  workerId: string;
   deviceId?: string;
-  scanner?: ScannerConnectionDTO;
+  scanner?: ControllerConnectionDTO;
```
Comment (Fabio1988) [inline packages/types/src/status.ts]: @jfberry should I also rename this variable? I checked, only DTOs internal used, exposed in API and UI only :) `controller` ?

File: packages/types/src/status.ts [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
   isAllocated: boolean;
-  mitm: MitmWorkerDTO;
-  workerId: string;
+  mitm: DeviceWorkerDTO;
```
Comment (Fabio1988) [inline packages/types/src/status.ts]: @jfberry should I also rename this variable? I checked, only DTOs internal used, exposed in API and UI only :) `deviceWorker` ?

File: packages/server/src/index.ts [hunk 3]
```diff
@@ -322,12 +328,15 @@ setInterval(() => {
   Object.entries(controlConnections).forEach(([, connection]) => {
     const origin = connection?.origin || 'Unknown';
     connectedDevices += 1;
+    const { memMitm, memFree, memStart } = connection.lastMemory;
 
-    const validMemFree = Number.isFinite(connection.lastMemory.memFree) ? connection.lastMemory.memFree : 0;
+    const validMemFree = valueOrZero(memFree);
```
Comment (na-ji) [inline packages/server/src/index.ts]: That's a good idea, but one of you will have to prepare for the conflict with #22

**Review by na-ji — APPROVED** [review]

File: packages/server/src/index.ts (hunk 3 above)
Comment (Fabio1988) [reply packages/server/src/index.ts]: Yeah I am prepared, fine to merge the other PR :)

**Pull request merged**